config/app.php

```diff
@@ -123,6 +123,8 @@
 
     'cipher' => 'AES-256-CBC',
 
+    'root_dir' => env('APP_ROOT_DIR', '/home/vagrant/Code/olm'),
```

Comment:
We already have checks for prod environment where this is used. It will run the same in every environment with this change. We should refactor this further but I was just getting annoyed by having to change it countless times.